src/main/java/org/examplesolid/domain/model/entity/CharacterEntity.java

```diff
@@ -1,26 +1,63 @@
 package org.examplesolid.domain.model.entity;
 
 import jakarta.persistence.*;
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.UUID;
 
-@Entity(name = "character_tv")
-@AllArgsConstructor
-@NoArgsConstructor
+import static org.examplesolid.domain.util.Constants.SEPARATOR_CONCAT_FUNFACT;
+
+@Entity
+@Table(name = "character_tv", uniqueConstraints = @UniqueConstraint(columnNames = "name"))
 @Getter
-@Setter
+@NoArgsConstructor
 public class CharacterEntity {
     @Id
     @GeneratedValue(strategy = GenerationType.UUID)
     private UUID uuid;
-    @Column(nullable = false, unique = true)
+    @Setter
+    @Column(nullable = false, unique = true, length = 100)
     private String name;
-    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.REMOVE)
-    private List<EpisodeEntity> episodes;
+    @Setter
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "character", cascade = CascadeType.ALL)
+    private Set<EpisodeEntity> episodes;
+    private String funFacts;
+
+    public void addFunFact(String funFact) {
+        if (this.funFacts == null) {
+            this.funFacts = funFact;
+        } else {
+            this.funFacts = funFacts.concat(SEPARATOR_CONCAT_FUNFACT).concat(funFact);
+        }
+    }
+
+    public CharacterEntity(UUID uuid, String name, String funFact) {
+        this.uuid = uuid;
+        this.name = name;
+        this.addFunFact(funFact);
+    }
+
+    public CharacterEntity(String name, String funFacts, Set<Integer> episodes) {
+        this.name = name;
+        this.funFacts = funFacts;
+        this.episodes = new HashSet<>();
+        for (Integer episode : episodes) {
```

Comment:
Quizas en lugar de iterar en la lista de episodios, se pueda usa el stream.map asi:
`List<EpisodeEntity> entities = episodes.stream().map(episode->new EpisodeEntity(episode, this)).collect(Collectors.toList());`